stable/osg-hosted-ce/osg-hosted-ce/README.md

```diff
@@ -267,6 +267,36 @@ To avoid these rate limits, it's possible to bootstrap the certificate request p
           HostKeySecret: <YOUR HOST KEY SECRET NAME>
           HostCertSecret: <YOUR HOST CERT SECRET NAME>
 
+### NetworkPolicy
+
+A generic NetworkPolicy for restricting traffic to and from the HostedCE. Allows configuration based on a list of IP CIDR ranges.
+
+By default NetworkPolicy is disabled and the application is open to all traffic.
+
+Network Policy can be thought of like a firewall for the application.
+
+Traffic can be restricted to only allow communication between the CE, OSG Factory, and your own cluster environment.
+
+```
+NetworkPolicy:
+  Enabled: false
+  AllowedCIDRs:
+  - 0.0.0.0/0
+```
```

Comment:
With the previous sentence, the following example should show the configuration that would restrict access between the OSG factories and the CE

@jdost321 @mmascher what're the factory IP ranges?